--- xgboost
+++ inaccel-xgboost
@@ -31,7 +31,7 @@
 namespace xgboost {
 enum class TreeMethod : int {
   kAuto = 0, kApprox = 1, kExact = 2, kHist = 3,
-  kGPUHist = 5
+  kGPUHist = 5, kFPGAExact = 6
 };

 // boosting process types
@@ -99,6 +99,7 @@
         .add_enum("exact",     TreeMethod::kExact)
         .add_enum("hist",      TreeMethod::kHist)
         .add_enum("gpu_hist",  TreeMethod::kGPUHist)
+        .add_enum("fpga_exact",     TreeMethod::kFPGAExact)
         .describe("Choice of tree construction method.");
   }
 };
